workspaces/github-discussions/plugins/github-discussions-common/src/permissions.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createPermission } from '@backstage/plugin-permission-common';
+
+export const githubDiscussionsDocumentReadPermission = createPermission({
+  name: 'search.discussions.read',
```

Comment:
so that we decouple it from search if in the future a backend plugin comes

```suggestion
export const githubDiscussionsReadPermission = createPermission({
  name: 'github.discussions.read',
```